src/html.js

```diff
@@ -17,6 +17,7 @@ export default class HTML extends React.Component {
         </head>
         <body {...this.props.bodyAttributes}>
           {this.props.preBodyComponents}
+          <noscript>This website requires JavaScript.</noscript>
```

Comment:
is there a reason for adding this in this PR? (it's totes fine just checking to make sure it's here on purpose!)

Reply:
Yep, its a check in google lighthouse and improves the score there. Something about fallback content.